File: TesteAndroidv2/app/src/main/java/com/everis/TesteAndroidv2/StatementService.java
package com.everis.TesteAndroidv2;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface StatementService {

    @GET("api/statements/{idUser}")
    Call<Lancamento> checarExtrato(@Path("idUser") String idUser);
}

File: bankapplication/app/src/main/java/br/com/crmm/bankapplication/framework/datasource/remote/abstraction/StatementService.kt
package br.com.crmm.bankapplication.framework.datasource.remote.abstraction

import br.com.crmm.bankapplication.framework.datasource.remote.dto.response.StatementListResponseDTO
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface StatementService {

    @GET("statements/{userId}")
    fun fetch(
        @Path("userId") shoppingListId: String
    ): Call<StatementListResponseDTO>
}